roles/bmc_fw_update/tasks/main.yml

```diff
@@ -4,14 +4,14 @@
 ---
 # tasks file for bmc_fw_update
 
-- name: Get Firmware Inventory
-  ansible.builtin.include_role:
-    name: get_bmc_facts
-  register: get_bmc_facts_before
```

Comment:
why this is commented out ?

Reply:
why do we need it? It is not being used. And when we update the bmc and the cec, the second time we get a meaningless get_bmc_facts_before (after the bmc and before the cec ...?)

Reply:
if we don't need it, remove it :-) don't keep comment out code